go.mod

```diff
@@ -274,6 +274,8 @@ replace (
 	github.com/grafana/grafana/pkg/apiserver => ./pkg/apiserver
 )
 
+require github.com/grafana/grafana/pkg/build v0.0.0-20250806181750-295ace108dd7
```

Comment:
this needs to be removed from go.mod and go.sum (and anywhere else it was added). not sure why this happens